python/pyspark/mllib/tests.py

```diff
@@ -76,7 +76,8 @@
     pass
 
 ser = PickleSerializer()
-sc = SparkContext('local[4]', "MLlib tests")
+conf = SparkConf().set("spark.driver.allowMultipleContexts", "true")
```

Comment:
@mengxr This was my (failing) attempt to correct the test errors (caused by ```--parallelism=4```?). Maybe @JoshRosen could comment?